ui/packages/app/grafana-datasource-plugin/.github/workflows/ci.yml

```diff
@@ -0,0 +1,75 @@
+name: CI
```

Comment:
Please ignore the GitHub workflow files in this child folder, I'll fix these and move them into the root `.github` folder in the next PR.